lib/fs-helpers.js

```diff
@@ -38,33 +39,74 @@ var helpers = {
     },
 
     /**
+     * Makes directory recursively.
+     *
+     * @param {String} dirPath
+     */
+    isAbsolutePathUnix: function (dirPath) {
+        return /^\//.test(dirPath);
+    },
+
+    /**
      * Checks existence.
      *
-     * @param {String} path
+     * @param {String} filepath
      */
     exists: typeof fs.access === 'function' ?
-        function (path) {
+        function (filepath) {
             try {
-                fs.accessSync(path);
+                fs.accessSync(filepath);
                 return true;
             } catch (err) {
                 return false;
             }
         } :
-        function (path) {
-            return fs.existsSync(path);
+        function (filepath) {
+            return fs.existsSync(filepath);
         },
 
     /**
-     * @param {String} path
+     * @param {String} filepath
      * @returns {Boolean}
      */
-    isExecutable: function (path) {
-        var stats = fs.lstatSync(path);
+    getFileStat: function (filepath) {
+        var stats = fs.lstatSync(filepath);
+
+        if (stats.isSymbolicLink()){
+            var linkPath = fs.readlinkSync(filepath);
+            linkPath = ((path.isAbsolute || helpers.isAbsolutePathUnix)(linkPath) ?
```

Comment:
It's a bad idea to rewrite the variable. It's better to create a new one.  `resolvedLinkPath`?